.circleci/config.yml

```diff
@@ -114,6 +114,21 @@ commands:
           destination: capybara
 
 jobs:
+  setup:
+    working_directory: ~/identity-idp
+    executor: ruby_browsers
+    steps:
+      - checkout
+      - bundle-yarn-install
+      - run:
+          name: Test Setup
+          command: |
+            bundle exec rake assets:precompile
+      - persist_to_workspace:
+          root: .
+          paths:
+            - public/assets
```

Comment:
I think we'll also want to persist `public/packs` (and `public/packs-test`)? These are Webpacker's precompiled output.

Reply:
Would there be an advantage to also include `node_modules` and `vendor/bundle` in the persisted paths, and then exclude `bundle-yarn-install` in `build` and `javascript_build`?

Reply:
packs added in 8f526c01a

>Would there be an advantage to also include node_modules and vendor/bundle in the persisted paths, and then exclude bundle-yarn-install in build and javascript_build?

It's worth a shot, but the time saved is probably minimal since cache restoration takes ~7 seconds (testing in b1bf6e3e5)

Reply:
I tried it, and the seconds saved didn't seem worth the additional complexity